sound/soc/sof/amd/acp.h

```diff
@@ -153,6 +164,9 @@ struct acp_dev_data {
 	struct acp_dsp_stream stream_buf[ACP_MAX_STREAM];
 	struct acp_dsp_stream *dtrace_stream;
 	struct pci_dev *smn_dev;
+
+	/* hw config */
+	const struct sof_amd_acp_desc *desc;
```

Comment:
do you really need this? can you not access desc via get_chip_info(sdev->pdata) already?

Reply:
Used the above suggestion  and implemented.